app/views/favorites/index.html.erb

```diff
@@ -1,13 +1,26 @@
+<h4><%= link_to "Adopt Your Favorite Pets", '/applications/new' %></h4>
 <% if @fav_pets.empty? %>
   <h3>You have not favorited any pets.</h3>
 <% else %>
   <%= button_to 'Remove All Pets from Favorites', "/favorites", method: :delete, class: 'button' %>
 <% end %>
 
+<section id="available">
 <% @fav_pets.each do |pet| %>
   <section id="favorite<%= pet.id %>">
     <%= image_tag(pet.image, {:style => "width:200px;"}) %>
-    <h4><a href="/pets/<%= pet.id %>">Name: <%= pet.name %></a></h4>
+     <h4><%= link_to "Name: #{pet.name}", href="/pets/#{pet.id}" %></h4>
     <%= button_to 'Remove this pet from favorites.', "/favorites/#{pet.id}", method: :delete, class: 'button' %>
   </section>
 <% end %>
+</section>
+
+
+
+<h2>Pending Applications</h2>
+
+<section id="pending-applications">
+  <% PetApplication.get_pets.each do |pet| %>
+    <h4><%= link_to "Name: #{pet.name}", href="/pets/#{pet.id}" %></h4>
+  <% end %>
+</section>
```

Comment:
Pending section needs revision stylistically, but also possibly with different code once we use an adopted boolean in pets model.